package.json

```diff
@@ -25,17 +26,19 @@
         "htmllint": "./node_modules/.bin/htmlhint --config .htmlhintrc.json src",
         "eslint": "./node_modules/.bin/eslint src",
         "tests": "mocha --require babel-register tests/test.js",
-        "cover": "node_modules/.bin/babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- tests/test.js"
+        "cover": "node_modules/.bin/babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- tests/test.js",
+        "eStart": "electron ."
     },
     "dependencies": {
         "@uirouter/angularjs": "1.0.15",
-        "angular": "1.6.9",
+        "angular": "^1.6.9",
```

Comment:
Please remove the carrot in front of the 1, we want to enforce exactly what version of each package we have.